### PR TITLE
Implement homework JS3 W2

### PR DESCRIPTION
**Pull request by valmdz**

### CODE REVIEW

File: javascript/javascript3/week2/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <link href="main.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,800" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+    <main>
+        <ul class="marks">
+            <li></li>
+            <li></li>
```
Comment (bashirmja) [inline javascript/javascript3/week2/index.html:12]: consider to delete empty list items and if you need them you can create them in js.

File: javascript/javascript3/week2/indexOne.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <script src="movies.js" defer></script>
+ <script src="resolveAfter.js" defer></script>
```
Comment (bashirmja) [inline javascript/javascript3/week2/indexOne.html:7]: consider to move js file to the end of the file before </body>

File: javascript/javascript3/week2/main.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+const boxes = [1, 2, 3].map((n) => ({
+  source: document.querySelector(`.marks li:nth-child(${n})`),
```
Comment (bashirmja) [inline javascript/javascript3/week2/main.js:2]: it is not the best practice to use nested child selector. consider using a class name.

File: javascript/javascript3/week2/movies.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json";
+
+async function getMovies() {
+  try {
+    const data = await fetch(urlMovies);
+    const movies = await data.json();
+
+    const badMovies = movies
+      .filter((movie) => movie.rating < 4)
+      .map((movies) => movies);
```
Comment (bashirmja) [inline javascript/javascript3/week2/movies.js:11]: is there any reason to use map here?

**Review by bashirmja — CHANGES_REQUESTED** [review]
Great job. I just added a few comments. consider them if they are relevant.

File: javascript/javascript3/week2/index.html (hunk 1 above)
Comment (valmdz) [reply javascript/javascript3/week2/index.html:12]: Hi Bashri, thank you.  We were given that file.

**Pull request merged**